search-lambda/CKS.SearchLambda/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Do we need this file? This implies it's for deployment via VS tools, but we're using TF to create this and the API gateway, so i think having this file could be confusing. Especially as it has a different region to the TF stuff

Reply:
add some better notes - re this is used by vs tools etc.....

Reply:
add info to readme about what this file does